tests/common/fixtures/advanced_reboot.py

```diff
@@ -382,6 +381,17 @@ def runRebootTestcase(self, prebootList=None, inbootList=None, prebootFiles=None
         # Handle mellanox platform
         self.__handleMellanoxDut()
```

Comment:
why is the `imageInstall` a wrapper around `__HandleMellanoxDut`? is there a code missing here?

Reply:
nevermind, I saw it being used below.